src/backend/el.d

```diff
@@ -0,0 +1,221 @@
+/**
+ * Compiler implementation of the
+ * $(LINK2 http://www.dlang.org, D programming language).
+ *
+ * Copyright:   Copyright (C) 1985-1995 by Symantec
+ *              Copyright (c) 1999-2016 by Digital Mars, All Rights Reserved
+ * Authors:     $(LINK2 http://www.digitalmars.com, Walter Bright)
+ * License:     backendlicense.txt
+ * Source:      $(DMDSRC backend/_el.d)
+ */
+
+module ddmd.backend.el;
+
+import ddmd.backend.cdef;
+import ddmd.backend.cc;
+import ddmd.backend.type;
+
+import ddmd.backend.cc : Symbol;
```

Comment:
That can't possibly be the cause of double-definition related errors.  Either they exist in master and you can import them, or they don't and there's no conflict.